### DISCUSSION TITLE
How can add different background color for different sidebar in docusaurus 2

### DISCUSSION THREAD

**Discussion by bhakti24-create** [facebook/docusaurus#3014] [General]
How can add different background color for different sidebar in docusaurus 2. Suppose I have two navbar links. On one link click should show orange background color to sidebar and on another link click should show yellow background color to sidebar.

**Comment by slorber** [comment]
Hi,

When you mean 2 sidebars, do you mean something like we have on D2 website? where the left sidebar nav changes according to the currently browsed doc?

Ex, these 2 docs have different sidebars
- https://v2.docusaurus.io/docs/next/contributing
- https://v2.docusaurus.io/docs/next/team


What's the purpose of this, to give more context to the user on which part of the doc he's on?

Do you have an online site that could help me understand better?
